inc/Mailer.class.php

```diff
@@ -183,7 +183,7 @@ public function setParameters($param) {
      *
      * @param string $text     plain text body
      * @param array  $textrep  replacements to apply on the text part
-     * @param array  $htmlrep  replacements to apply on the HTML part, leave null to use $textrep
+     * @param array  $htmlrep  replacements to apply on the HTML part. When leave null the replacements from $textrep will be used but all the urls will be wrapped with <a> tag. Ex. <a href="http://bar.com">http://bar.com</a>.
```

Comment:
That is quite long. How about `replacements to apply on the HTML part, null to use $textrep (with urls wrapped in <a> tags)`?